addons/account/views/account_move_views.xml

```diff
@@ -1087,7 +1087,7 @@
                                         <field name="invoice_outstanding_credits_debits_widget"
                                             class="oe_invoice_outstanding_credits_debits"
                                             colspan="2" nolabel="1" widget="payment"
-                                            attrs="{'invisible': ['|', ('state', '!=', 'posted'), ('move_type', 'in', ('out_receipt', 'in_receipt'))]}"/>
```

Comment:
Since this line hasn't changed since 2019, I would say we didn't think about this use case in PR #86394.